client/mysqlbinlog.cc

```diff
@@ -1458,6 +1461,7 @@ Exit_status process_event(PRINT_EVENT_INFO *print_event_info, Log_event *ev,
   retval= ERROR_STOP;
 end:
   rec_count++;
+  last_processed_datetime = last_ev_when;
```

Comment:
> Or perhaps we don’t need it at all?

We do need the variable because in the case of `FORMAT_DESCRIPTION_EVENT`, the `ev` pointer is nullified. 

Though I agree that `last_` as a prefix no longer makes sense (it was left-over from the original implementation). 

Note too, the formatting here is still incorrect:

```suggestion
  last_processed_datetime= ev_when;
```